scannermsbuild.properties

```diff
@@ -6,8 +6,8 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-msbuild/
 issueTrackerUrl=https://github.com/SonarSource/sonar-scanner-msbuild/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-msbuild
-archivedVersions=4.7,4.7.1,4.8,4.9,4.10,5.0,5.0.3,5.0.4,5.1,5.2,5.2.1,5.2.2,5.3.1,5.3.2,5.4,5.4.1,5.5.0,5.5.1,5.5.2,5.5.3,5.6.0,5.7.0,5.7.1,5.7.2,5.8.0,5.9.0,5.9.1
-publicVersions=5.9.2
+archivedVersions=4.7,4.7.1,4.8,4.9,4.10,5.0,5.0.3,5.0.4,5.1,5.2,5.2.1,5.2.2,5.3.1,5.3.2,5.4,5.4.1,5.5.0,5.5.1,5.5.2,5.5.3,5.6.0,5.7.0,5.7.1,5.7.2,5.8.0,5.9.0,5.9.1,5.9.2
+publicVersions=5.10.0
```

Comment:
To keep it simple, we generally don't put the patch digit when it's .0.

Reply:
Updated, thanks